### DISCUSSION TITLE
is there a way to disable the analytics part?

### DISCUSSION THREAD

**Discussion by ghost** [wiredashio/wiredash-sdk#363] [Q&A] [answered]
i'm just interested in the feedback features. is there a way to disable the analytics?

**Comment by passsy** [comment, marked as answer]
Thanks for reaching out ✌️

Our pricing is based on the monthly active devices of your app. Therefore, it is required that every app pings our server regularly. These pings are fully anonymous, and you can learn more about our data collection practices at https://docs.wiredash.com/guide/analytics/privacy.

While it's technically possible to disable any analytics requests, doing so would require a custom business plan. Projects with feedback but without monthly active devices trigger our fraud detection system.

We’d love to hear more about your use case! Feel free to contact us at `hey@wiredash.com` to discuss potential solutions.
